packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -7,17 +7,20 @@ import React, {
 } from "react";
 import classnames from "classnames";
 import { Clearable, useShowClear } from "@jobber/hooks/useShowClear";
-import { AnimatePresence, motion } from "framer-motion";
 import { tokens } from "@jobber/design";
 import { FormFieldProps } from "./FormFieldTypes";
 import styles from "./FormField.module.css";
 import { AffixIcon, AffixLabel } from "./FormFieldAffix";
 import { FormFieldDescription } from "./FormFieldDescription";
 import { ClearAction } from "./components/ClearAction";
-import { useToolbar } from "./hooks/useToolbar";
 import { useFormFieldFocus } from "./hooks/useFormFieldFocus";
-import { useIsSafari } from "./hooks/useIsSafari";
 import { InputValidation } from "../InputValidation";
+import { isFirefox } from "../utils/getClientBrowser";
+
+const IS_FIREFOX = isFirefox(navigator.userAgent);
```

Comment:
I think isFirefox at the top level may break on SSR (existing remix stuff) because I am 99% sure navigator is undefined on the server